specification/migrate/resource-manager/Microsoft.Migrate/AssessmentProjects/preview/2023-04-01-preview/migrate.json

```diff
@@ -0,0 +1,15310 @@
+            "pattern": "^[^<>&:\\?/#]{1,260}$"
+          },
+          {
+            "name": "businessCaseName",
+            "in": "path",
+            "description": "Business case ARM name",
+            "required": true,
+            "type": "string",
+            "pattern": "^[^<>&:\\?/#]{1,260}$"
+          },
+          {
+            "name": "avsSummaryName",
+            "in": "path",
+            "description": "Gets the Name of the Avs Summary.",
+            "required": true,
+            "type": "string",
+            "pattern": "^[^<>&:\\?/#]{1,260}$"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "ARM operation completed successfully.",
+            "schema": {
+              "$ref": "#/definitions/AvsSummary"
+            }
+          },
+          "default": {
+            "description": "An unexpected error response.",
+            "schema": {
+              "$ref": "../../../../../../common-types/resource-management/v5/types.json#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "BusinessCaseAvsSummaryOperations_Get_MaximumSet_Gen": {
+            "$ref": "./examples/BusinessCaseAvsSummaryOperations_Get_MaximumSet_Gen.json"
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Migrate/assessmentProjects/{projectName}/businesscases/{businessCaseName}/compareSummary": {
+      "post": {
+        "operationId": "BusinessCaseOperations_CompareSummary",
+        "tags": [
+          "BusinessCaseOperations"
+        ],
+        "summary": "Get Compare summary report for Business case.",
+        "description": "A long-running resource action.",
+        "parameters": [
+          {
+            "$ref": "../../../../../../common-types/resource-management/v5/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "../../../../../../common-types/resource-management/v5/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../../../../common-types/resource-management/v5/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "name": "projectName",
+            "in": "path",
+            "description": "Assessment Project Name",
+            "required": true,
+            "type": "string",
+            "pattern": "^[^<>&:\\?/#]{1,260}$"
+          },
+          {
+            "name": "businessCaseName",
+            "in": "path",
+            "description": "Business case ARM name",
+            "required": true,
+            "type": "string",
+            "pattern": "^[^<>&:\\?/#]{1,260}$"
+          },
+          {
+            "name": "body",
+            "in": "body",
+            "description": "The content of the action request",
+            "required": true,
+            "schema": {
+              "type": "object"
```

Comment:
What is the input here? Considering the operation is meant for comparing, there will be a second input. If there is an input, have a corresponding definition of the input structure.